libr/core/cmd_anal.c

```diff
@@ -7715,7 +7715,8 @@ static int cmd_anal_all(RCore *core, const char *input) {
 					goto jacuzzi;
 				}
 				if (r_config_get_i (core->config, "anal.autoname")) {
-					oldstr = r_print_rowlog (core->print, "Constructing a function name for fcn.* and sym.func.* functions (aan)");
+					oldstr = r_print_rowlog (core->print, "Speculatively constructing a function name "
+					                         "for fcn.* and sym.func.* functions (aan)");
 					r_core_anal_autoname_all_fcns (core);
```

Comment:
I don't think it should be under "aan" because "aan" might have a use and has its own logic to auto-name functions by used flags. I believe "aan" should be an independent command outside "aaa" and analyze the jmp-thunk-table inside "aaa" regardless of "anal.autoname".

Reply:
well. i think that this is actually an autonaming hack, what we should probably do is convert anal.autoname into 'int' instead of 'bool' and allow to define the level of aggressivity on autonaming things

Reply:
agree, though the process of defining the levels of aggressivity might be a political nightmare